domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/RawEntity.java

```diff
@@ -18,24 +18,50 @@
 
 import java.util.Map;
 
+/**
+ * The {@code RawEntity} class represents a row of a GTFS file as raw string data.
+ */
 public class RawEntity {
 
+    /**
+     * Key: header string from original CSV file
+     * Value: a String, the raw value read from the CSV file
+     */
     private final Map<String, String> contentByHeaderMap;
     private final int entityIndex;
 
+    /**
+     * @param contentByHeaderMap the object mapping rows of a GTFS .txt file on header name
+     * @param entityIndex        the 1 based index of the row
+     */
     public RawEntity(Map<String, String> contentByHeaderMap, int entityIndex) {
         this.contentByHeaderMap = contentByHeaderMap;
         this.entityIndex = entityIndex;
     }
 
+    /**
+     * Returns the value contained in a row for a given header (column)
+     *
+     * @return the value contained in a row for a given header (column)
+     */
     public String get(final String header) {
         return contentByHeaderMap.get(header);
     }
 
+    /**
+     * Returns the number of headers (columns)
+     *
+     * @return the number of headers (columns)
+     */
     public int size() {
         return contentByHeaderMap.size();
     }
 
+    /**
+     * Returns the 1 based index of the row
+     *
+     * @return the 1 based index of the row
```

Comment:
Same as above - please add documentation where this method is called from that it's 1-based.